code/game/machinery/seed_extractor.dm

```diff
@@ -8,16 +8,53 @@
 
 /obj/structure/machinery/seed_extractor/attackby(obj/item/O as obj, mob/user as mob)
 
+	// Plant bag and other storage containers.
+	if(istype(O,/obj/item/storage))
+		var/obj/item/storage/container = O
+		if(container.contents.len > 0)
```

Comment:
length(container.contents) over .len since its nullsafe (and tiny bit faster too, at least I was told so.)